### PR TITLE
SilverlakeSoftwareLLC.Velocity version 1.2.1.20436

### PR DESCRIPTION
**Pull request by sadmanca**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15784)

### CODE REVIEW

File: manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: SilverlakeSoftwareLLC.Velocity
+PackageVersion: 1.2.1.20436
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.installer.yaml:11]:
```suggestion
  Architecture: x86
```

File: manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SilverlakeSoftwareLLC.Velocity
+PackageVersion: 1.2.1.20436
+PackageLocale: en-US
+Publisher: Silverlake Software LLC
+PackageName: Velocity
+License: Copyright © 2021 Silverlake Software LLC
```
Comment (ItzLevvie) [inline manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.locale.en-US.yaml:8]:
```suggestion
License: Copyright © 2021 Silverlake Software LLC. All Rights Reserved.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
`PackageIdentifier` should be `SilverlakeSoftware.Velocity`  instead of `SilverlakeSoftwareLLC.Velocity`

File: manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: SilverlakeSoftwareLLC.Velocity
```
Comment (ItzLevvie) [inline manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.installer.yaml:3]:
```suggestion
PackageIdentifier: SilverlakeSoftware.Velocity
```
You'll also need to change the folder name and its files from `SilverlakeSoftwareLLC` to `SilverlakeSoftware` using GitHub's desktop client on Windows.

File: manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SilverlakeSoftwareLLC.Velocity
```
Comment (ItzLevvie) [inline manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: SilverlakeSoftware.Velocity
```
You'll also need to change the folder name and its files from `SilverlakeSoftwareLLC` to `SilverlakeSoftware` using GitHub's desktop client on Windows.

File: manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: SilverlakeSoftwareLLC.Velocity
```
Comment (ItzLevvie) [inline manifests/s/SilverlakeSoftwareLLC/Velocity/1.2.1.20436/SilverlakeSoftwareLLC.Velocity.yaml:3]:
```suggestion
PackageIdentifier: SilverlakeSoftware.Velocity
```
You'll also need to change the folder name and its files from `SilverlakeSoftwareLLC` to `SilverlakeSoftware` using GitHub's desktop client on Windows.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**